### PR TITLE
Fix broken references in README.md

### PR DESCRIPTION
**Pull request by DCSBL**
### Description of the feature or fix
The README on the main GitHub page had some broken or outdated links to the documentation 

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md) > **Is this required for a change like this?**
- [x] Update the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,14 +115,14 @@ LVGL is also avaiable as:
 This list shows the recommended way of learning the library:
 1. Check the [Online demos](https://lvgl.io/demos) to see LVGL in action (3 minutes)
 2. Read the [Introduction](https://docs.lvgl.io/master/intro/index.html) page of the documentation (5 minutes)
-3. Get familiar with the basics on the [Quick overview](https://docs.lvgl.io/master/html/get-started/quick-overview.html) page (15 minutes)
-4. Set up a [Simulator](https://docs.lvgl.io/latest/master/get-started/pc-simulator.html) (10 minutes)
+3. Get familiar with the basics on the [Quick overview](https://docs.lvgl.io/master/get-started/quick-overview.html) page (15 minutes)
+4. Set up a [Simulator](https://docs.lvgl.io/master/get-started/pc-simulator.html) (10 minutes)
 5. Try out some [Examples](https://github.com/lvgl/lvgl/tree/master/examples)
-6. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/master/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_&type=&language=)
```
Comment (DCSBL) [inline README.md:121]: This one is just for style.

File: README.md [hunk 2]
```diff
@@ -115,14 +115,14 @@ LVGL is also avaiable as:
...
+6. Port LVGL to a board. See the [Porting](https://docs.lvgl.io/master/porting/index.html) guide or check the ready to use [Projects](https://github.com/lvgl?q=lv_port_)
```
Comment (DCSBL) [inline README.md:121]: This one is just for style.

**Comment by kisvegabor** [comment]
Thank you ! :)

**Comment by embeddedt** [comment]
Thanks!

> CHANGELOG.md > Is this required for a change like this?

It's not really necessary for docs changes, as that would make it longer than it needs to be, and it can already get quite long. :wink:

Soon we are hoping to have the changelog be generated automatically, so that step shouldn't be required at all in the future.

**Pull request merged**